bodo/tests/conftest.py

```diff
@@ -800,41 +896,135 @@ def pytest_collect_file(parent, file_path: Path):
         )
 
 
-def get_tabular_connection(tabular_credential: str):
-    return (
-        "https://api.tabular.io/ws",
-        os.getenv("TABULAR_WAREHOUSE", "Bodo-Test-Iceberg-Warehouse"),
-        os.getenv("TABULAR_CREDENTIAL"),
+@pytest.fixture(scope="session")
+def polaris_package():
+    """
+    Ensure that the polaris client is installed
+    """
+    # Install the polaris client if it is not already installed
+    # This is a temporary solution until the polaris client is published
+    # https://github.com/apache/polaris/issues/19
+    try:
+        subprocess.run(["pip", "show", "polaris.management"], check=True)
+    except subprocess.CalledProcessError:
+        subprocess.run(
+            [
+                "pip",
+                "install",
+                "git+https://github.com/apache/polaris.git#subdirectory=regtests/client/python",
+            ]
+        )
+
+
+@pytest.fixture(scope="session")
+def polaris_token(polaris_server, polaris_package):
```

Comment:
Can we move these to the `test_iceberg/conftest.py` file? Don't wan't to clutter the top one too much for Iceberg-specific fixtures